docs/src/load_data.md

```diff
@@ -20,28 +18,9 @@ load_timeseries_data
 using ClustForOpt
 state="GER_1"
 # laod ts-input-data
-ts_input_data, = load_timeseries_data("CEP", state; K=365, T=24)
+ts_input_data = load_timeseries_data("CEP", state; K=365, T=24)
```

Comment:
- load_timeseries_data has only one output left
- we have to discuss how we shall do the data handling, examples